### PR TITLE
chore: run gofmt -s -l -w . for Go 1.19

### PR DESCRIPTION
**Pull request by quartzmo**

### CODE REVIEW

**Review by codyoss — APPROVED** [review]

File: google-api-go-generator/gen_test.go [hunk 1]
```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build go1.19
```
Comment (codyoss) [inline google-api-go-generator/gen_test.go:5]: Why is this needed?

Comment (quartzmo) [reply google-api-go-generator/gen_test.go:5]: The goldens for google-api-go-generator were failing for 1.19, so I updated them, and now the new goldens fail for the earliest version (and any other version < 1.19).

Comment (codyoss) [reply google-api-go-generator/gen_test.go:5]: Ah sure, since they include godoc. Interesting, would have never thought of that. Thanks!
